XcodeServerSDK/API Routes/XcodeServer+Bot.swift

```diff
@@ -50,8 +50,12 @@ extension XcodeServer {
                     return
                 }
                 
-                //we have platforms, find the one in the bot config and replace it
-                self.replacePlaceholderPlatformInBot(botOrder, platforms: platforms!)
+                do {
+                    //we have platforms, find the one in the bot config and replace it
+                    try self.replacePlaceholderPlatformInBot(botOrder, platforms: platforms!)
+                } catch {
+                    completion(response: .Error(error: error))
```

Comment:
You also need a return otherwise it would keep going and try to create the Bot anyway.